doc/api.md

```diff
@@ -43,6 +44,21 @@ var options = {
 var api = new JitsiMeetExternalAPI(domain, options);
 ```
 
+You can set the initial media devices for the call:
+
+```javascript
+var domain = "meet.jit.si";
```

Comment:
it's 2019, all browsers support `const` 😎 

Also, single quotes!

Reply:
Since I didn't want to change only the new examples but all of them I created another PR for this - https://github.com/jitsi/jitsi-meet/pull/4045 . @saghul could you please review?